src/int.cr

```diff
@@ -254,6 +254,10 @@ struct Int
     end
   end
 
+  def to_hex
```

Comment:
I'm not sure I fully follow what this does, but I don't think to_hex is a good name for a public API for it. Hexadecimal is about representation, but this doesn't return a representation, it returns a new value in the current representation (the `Int` subclass, which is also a representation of a number). From the name I would expect it to return the `Char`s `'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'` (or their lowercase counterparts) and raise for values > 15.